x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/empty_prompt/empty_prompt.tsx

```diff
@@ -63,21 +63,34 @@ export const AnalyticsEmptyPrompt: FC<Props> = ({ disabled, onCreateFirstJobClic
           <p>
             <FormattedMessage
               id="xpack.ml.overview.analyticsList.emptyPromptText"
-              defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis and put the annotated data in a new index. The classification and regression trained models can also be used for inference in pipelines and aggregations."
+              defaultMessage="Build and train outlier detection, regression, or classification machine learning models using data frame analytics."
             />
           </p>
           <EuiCallOut
             size="s"
             title={
               <FormattedMessage
                 id="xpack.ml.overview.analyticsList.emptyPromptHelperText"
-                defaultMessage="Data frame analytics requires specifically structured source data. Use {transforms} to create data frames before you create the jobs."
+                defaultMessage="Before building a data frame analytics job, use {transforms} to construct an {sourcedata} to analyze."
```

Comment:
Nit, but IMO "to analyze" is unnecessary here:
```suggestion
                defaultMessage="Before building a data frame analytics job, use {transforms} to construct an {sourcedata}."
```